tests/zquantum/core/wip/evolution_test.py

```diff
@@ -0,0 +1,200 @@
+import cirq
+import numpy as np
+import pytest
+import sympy
+from pyquil.paulis import PauliSum, PauliTerm
+
+from zquantum.core.utils import compare_unitary
+from zquantum.core.wip import circuits
+from zquantum.core.wip.evolution import (
+    generate_circuit_sequence,
+    time_evolution,
+    time_evolution_derivatives,
+    time_evolution_for_term,
+)
+
+PAULI_STRING_TO_CIRQ_GATE = {"XX": cirq.XX, "YY": cirq.YY, "ZZ": cirq.ZZ}
+
+
+def _cirq_exponentiate_term(term, qubits, time, order):
```

Comment:
I would change `order` to `trotter_order` so it's more explicit.

Reply:
Addressed in 8974b5a